app/winapp/rtknavi/naviopt.cpp

```diff
@@ -533,6 +541,11 @@ void __fastcall TOptDialog::SetOpt(void)
 	PrcOpt.tropopt   =TropOpt     ->ItemIndex;
 	PrcOpt.sateph    =SatEphem    ->ItemIndex;
 	PrcOpt.modear    =AmbRes      ->ItemIndex;
+	PrcOpt.wlmodear=0;
```

Comment:
nit, this line doesn't seem to be needed. prcopt.wlmodear has been initialised as 0 in https://github.com/swift-nav/RTKLIB/blob/fcb4097b272cc682a71f20e2bb5a31e2e098081d/src/rtkcmn.c#L207